acme_diags/parameter/meridional_mean_2d_parameter.py

```diff
@@ -1,11 +1,32 @@
 import numpy
 from .core_parameter import CoreParameter
-
+from acme_diags.driver.utils.general import strictly_increasing,strictly_decreasing,non_increasing,non_decreasing,monotonic
 
 class MeridionalMean2dParameter(CoreParameter):
     def __init__(self):
         super(MeridionalMean2dParameter, self).__init__()
-        self.plevs = numpy.logspace(2.0, 3.0, num=17)
+        self.plevs = numpy.logspace(2.0, 3.0, num=17).tolist()
```

Comment:
Does changing this to a `list` cause any problems?

Reply:
plevs being a numpy array caused the problem, then I converted it list. It doesn't show problem with testing